script/src/provider/dropbox.js

```diff
@@ -48,7 +48,7 @@ var Dropbox = function(){
             }
         };
 
-        dialog.setText("DROPBOX ://BassoonTracker is not yet connected to DropBox//Do you want to do that now?//(Bassoontracker will only have access/to its own Bassoontracker folder)");
+        dialog.setText("DROPBOX ://BassoonTracker is not yet connected to DropBox//Do you want to do that now?//(BassoonTracker will only have access//to its own BassoonTracker folder)");
```

Comment:
Careful, I'm adding a `/` because I understand it's necessary for a newline. Let me know if I'm wrong.

Reply:
Yes a / is a newline. 
// is 2 new lines so in this case I changed it back to 1 / ;-)

Reply:
Understood, thanks for the tip!